boms/zero-bom/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud</groupId>
+  <artifactId>zero-bom</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Google Cloud Platform Supported Libraries</name>
+  <description>
+    A compatible set of Google Cloud open source libraries.
+  </description>
+
+  <organization>
+    <name>Google LLC</name>
+    <url>https://cloud.google.com</url>
+  </organization>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <google.cloud.java.version>0.59.0-alpha</google.cloud.java.version>
+    <io.grpc.version>1.14.0</io.grpc.version>
+    <protobuf.version>3.6.1</protobuf.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <!-- GRPC; specifically https://github.com/grpc/grpc-java -->
```

Comment:
we should include _all_ grpc sub-modules, e.g. grpc-context, grpc-all, etc.

Reply:
These are the three that grpc recommends for using grpc in a conventional way. In contrast, we are creating a BOM here. So, anyone who *does* declare a dependency on modules outside of these three needs them to stay in sync with these three, otherwise everything breaks. Thus, all grpc modules should be in the BOM.